### DISCUSSION TITLE
exclude parameter in tar_watch()

### DISCUSSION THREAD

**Discussion by gorkang** [ropensci/targets#456] [Ideas]
Would it be possible to implement the `exclude` parameter of `tar_visnetwork()` in `tar_watch()`?

With `tar_visnetwork()` we can exclude targets very easily, for example: 
`tar_visnetwork(targets_only = TRUE, exclude = "annoying_targets")`

It would be great to be able to have an interactive version of the above using `tar_watch()`: 
`tar_watch(targets_only = TRUE,  exclude = "annoying_targets")`

Thanks!

**Comment by wlandau** [comment]
Sure, that sounds fine. Please give 3c83265c26f12a3efbcb76a1e4eba2c5dca37d50  a try. 

NB `tar_visnetwork()` supports tidyselect for `exclude`, but `tar_watch()` does not because of the various layers of exported functions that would otherwise have to manage quasiquotation.

**Reply by gorkang** [reply]
Works great, thanks a lot!
